React/Views/RCTView.m

```diff
@@ -1563,4 +1564,28 @@ - (BOOL)performDragOperation:(id <NSDraggingInfo>)sender
 }
 #endif // ]TODO(macOS ISS#2323203)
 
+#pragma mark - Keyboard Events
+
+#if TARGET_OS_OSX // TODO: add iOS keyboard event handling
```

Comment:
Is this TODO supposed to be tracked or more of a comment for readers?

Reply:
More of a comment. We have no business need for it, but if one day someone needed to add it then this would be where to start